### PR TITLE
Adding reusable redis PUBSUB scripts

### PR DESCRIPTION
**Pull request by chawlanikhil24**
So, I converted the redis pubsub smallfile code to reusable, python PUBSUB code, which can executed with any available benchmarks in ripsaw.

### CODE REVIEW

**Comment by chawlanikhil24** [comment]
@jtaleric @aakarshg @dustinblack

**Comment by jtaleric** [comment]
This code looks good for the starting trigger.

File: redis_scripts/publisher.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+
+def run_publisher(redis_host, redis_port, benchmark, pod_count):
+    try:
+        r = redis.StrictRedis(host=redis_host, port=redis_port)
+        count = 0
+        while count != pod_count:
+            redis_command = "PUBSUB NUMSUB {}".format(benchmark)
+            count = r.execute_command(redis_command)[1]
+            sys.stdout.write(str(count)+"\n")
```
Comment (jtaleric) [inline redis_scripts/publisher.py]: sys.stdout.write ? Curious why you are using this?

**Comment by chawlanikhil24** [comment]
changed sys.stdout.write() with print()

**Pull request merged**